ps5Controller/ps5Controller/HomeModule/HomeViewController.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+
+protocol HomeViewInterface {
+    func configureInitialView()
+    func reloadData()
+    func animateWhenReloadData()
+}
+
+final class HomeViewController: UIViewController {
+    @IBOutlet weak private var collectionView: UICollectionView!
+    @IBOutlet var tabButtons: [UIButton]!
+    var presenter: HomePresenter?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        presenter?.notifyViewLoaded()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        presenter?.notifyViewDidAppear()
+    }
+    
+    @IBAction func tabButtonsTapped(_ sender: UIButton) {
+        guard let index = tabButtons.firstIndex(of: sender),
```

Comment:
```
@IBAction func tabButtonsTapped(_ sender: UIButton) { 
presenter. tabButtonsTapped(index: tabButtons.firstIndex(of: sender))
}
```

bu kodun bu kadar olmalı. Diğer her şeyi taşımalısın

Reply:
bu mesajım hala geçerli. Buranın kodu üstte attığım kadar olmalı. bu kod bloğunda sadece
`presenter. tabButtonsTapped(index: tabButtons.firstIndex(of: sender))` olması gerekiyor.